lib/hooks/queries/usePortfolioPositions.ts

```diff
@@ -427,13 +434,38 @@ export const usePortfolioPositions = (
         color = "#DF0076";
       }
 
+      const totalCost = transactionHistory
+        .filter((transaction) => {
+          const scalarTransaction = transaction?.assetId["ScalarOutcome"];
```

Comment:
I think there are some utils in the new sdk for this sort of stuff. @yornaath could you point us in the right direction?